### DISCUSSION TITLE
Scaling down

### DISCUSSION THREAD

**Discussion by shashanksdixit** [kedacore/keda#1625] [Q&A / Need Help] [answered]
I have created the Azure service bus scaled object and I am able to scale my queue receiver pod using Keda scaler. The minimum number of pods are zero. So pods will get terminated if there are no messages in the queue. Once the pods are scaled, I checked that all the pods are getting terminated. But it takes around 10-15 mins. Is this period we can control? E.g. if we can mention the scaling down strategy is to check if there are no messages for lets say 5 mins or so.

**Comment by tomkerkhove** [comment, marked as answer]
Yes you can control this, it's documented on https://keda.sh/docs/2.1/concepts/scaling-deployments/#details

**Reply by shashanksdixit** [reply]
Thanks @tomkerkhove

**Reply by tomkerkhove** [reply]
Happy to help!
